glusterd2/volume/struct.go

```diff
@@ -80,20 +80,19 @@ type Subvol struct {
 
 // Volinfo repesents a volume
 type Volinfo struct {
-	ID          uuid.UUID
-	Name        string
-	Type        VolType
-	Transport   string
-	DistCount   int
-	Options     map[string]string
-	State       VolState
-	Checksum    uint64
-	Version     uint64
-	Subvols     []Subvol
-	Auth        VolAuth // TODO: should not be returned to client
-	GraphMap    map[string]string
-	HealEnabled bool
-	Metadata    map[string]string
+	ID        uuid.UUID
+	Name      string
+	Type      VolType
+	Transport string
+	DistCount int
+	Options   map[string]string
+	State     VolState
+	Checksum  uint64
+	Version   uint64
+	Subvols   []Subvol
+	Auth      VolAuth // TODO: should not be returned to client
```

Comment:
Unrelated to your PR: The TODO comment is no longer valid. As you will be refreshing this PR anyway to address other comments, you can remove it too :)